test/spec/unit/bootstraps/liveblogTest.js

```diff
@@ -140,7 +140,6 @@ define([
             it('sets up scroll event listener', function () {
                 liveblog.init();
 
-                expect(window.addEventListener).to.have.been.calledOnce;
```

Comment:
I don't understand the value of now changing this test to check whether or not its been called twice. That is why I just removed it, rather than replace